tests/system/Helpers/Url_helper_test.php

```diff
@@ -0,0 +1,79 @@
+<?php
```

Comment:
Oops! I just pushed some bug-fixes to the base_url and site_url that had a test file, also - and it's named slightly differently. Sorry.